dist/leaflet.distortableimage.css

```diff
@@ -1,3 +1,9 @@
+body {
```

Comment:
we have started using more specific css selectors in our library so they don't overwrite anything downstream. see #330